app/controllers/apartments_controller.rb

```diff
@@ -1,2 +1,5 @@
 class ApartmentsController < ApplicationController
+
+    def index
+        @apartments = Apartments.all
```

Comment:
Seems like this code is on the wrong branch or if you want it here it's incomplete.